doc/sphinx/reference/varnishd.rst

```diff
@@ -461,6 +461,7 @@ specific options. Available jails are:
 
   - It warns when *workdir* is not on a ``tmpfs``.
   - It tries to keep the process dumpable after dropping privileges.
+  - It tries to disable Transparent Hugepage in ``varnishd``
```

Comment:
Instead of just adding a bullet point here, please also document the `transparent_hugepage` parameter below, like the UNIX jail documentation. In particular, document that failing to enable or disable THP is not a fatal error.

Also, please apply this change a few lines above:

```diff
--j <linux[,user=`user`][,ccgroup=`group`][,workuser=`user`]>
+-j <linux[,transparent_hugepage=`action`][,<unix jail option>...]>
```

Reply:
Maybe add " by default." at the end of the sentence. At the very least, please add a dot to be consistent with @nigoroll's style change for the bullet points above.